src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -0,0 +1,582 @@
+<template>
+  <div
+    v-if="isComponentExists"
+    class="va-slider d-flex align--center"
+    :class="sliderClass"
+    @click="wrapClick"
+  >
+    <span
+      v-if="label && !invertLabel"
+      class="va-slider__label title">
+      {{ label }}
+    </span>
+    <span
+      v-if="icon"
+      class="va-slider__label title">
+      <va-icon :icon="icon" :color="color" :size="16"/>
+    </span>
+    <div
+      class="va-slider__container d-flex align--center"
+      ref="elem"
+    >
+      <div
+        class="va-slider__container__track"
+        :style="{ backgroundColor: color }"/>
+      <template v-if="pins">
+        <div
+          v-for="(pin, key) in pinsCol"
+          :key="key"
+          class="va-slider__container__mark"
+          :class="{ 'va-slider__container__mark--active': isRange ? pin * step > val[0] && pin * step < val[1] : pin * step < val}"
+          :style="{ left: pin * step + '%' }"
+        />
+      </template>
+      <template v-if="isRange">
+        <div
+          ref="process"
+          class="va-slider__container__track va-slider__container__track--active"
+          :style="processStyles"
+          @mousedown="moveStart($event, 0)"/>
+        <div
+          ref="dot0"
+          class="va-slider__container__handler"
+          :style="dotStyles[0]"
+          @mousedown="moveStart($event, 0)"
+        >
+          <div
+            v-if="valueVisible"
+            class="va-slider__container__handler-value title"
+          >
+            {{ val[0] }}
+          </div>
+        </div>
+        <div
+          ref="dot1"
+          class="va-slider__container__handler"
+          :style="dotStyles[1]"
+          @mousedown="moveStart($event, 1)"
+        >
+          <div
+            v-if="valueVisible"
+            class="va-slider__container__handler-value title">
+            {{ val[1] }}
+          </div>
+        </div>
+      </template>
+      <template v-else>
+        <div
+          ref="process"
+          class="va-slider__container__track va-slider__container__track--active"
+          :style="processStyles"
```

Comment:
use processedStyles or computedStyles for variable name. not process. this applies to all variables in the pr